arches/install/arches-templates/project_name/templates/email/package_load_complete_email_notification.htm

```diff
@@ -0,0 +1,30 @@
+{% load static %}
+{% load template_tags %}
+{% load i18n %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        {% include 'custom_email_css.htm' %}
+    </head>
+
+    <body>
+
+        {% include 'custom_email_header.htm' %}
+
+        {% if username != "" %}
+            <p>Dear {{username}},</p>
```

Comment:
same thing here about parameterizing the salutation

Reply:
@chrabyrd Changed as requested